DOCSUP-11968: Document the SQL-statement EXPLAIN ESTIMATE (#27536)
ClickHouse/ClickHouse: docs/en/sql-reference/statements/explain.md

            (ReadFromStorage)
            NumbersMt × 2 0 → 1
```
### EXPLAIN ESTIMATE {#explain-estimate}

> Shows information about read rows, marks and parts from [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md#table_engines-mergetree) tables. 

Reviewer: ```suggestion
Shows information about rows, marks and parts to be read from [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md#table_engines-mergetree) tables. 
```

> Shows information about read rows, marks and parts from [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md#table_engines-mergetree) tables. 

Reviewer: ```suggestion
Shows the estimated number of rows, marks and parts to be read from [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md#table_engines-mergetree) tables while processing the query. 
```

> Example:

Reviewer: ```suggestion
**Example**
```
